db-engine/src/main/java/com/splicemachine/db/iapi/stats/ColumnStatisticsMerge.java

```diff
@@ -112,9 +110,12 @@ public void accumulate(ColumnStatisticsImpl value) throws StandardException {
     @Override
     public void merge(ColumnStatisticsMerge otherAggregator) {
         ColumnStatisticsMerge columnStatisticsMerge = (ColumnStatisticsMerge) otherAggregator;
-        quantilesSketchUnion.update(columnStatisticsMerge.quantilesSketchUnion.getResult());
-        frequenciesSketch.merge(columnStatisticsMerge.frequenciesSketch);
-        thetaSketchUnion.update(columnStatisticsMerge.thetaSketchUnion.getResult());
+        if (!columnStatisticsMerge.quantilesSketchUnion.getResult().isEmpty()) {
```

Comment:
Could the input param otherAggregator be null? Maybe we need to be more defensive here.